ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/workspace/WorkspaceEventsNotifier.java

```diff
@@ -298,7 +314,7 @@ public void apply(final List<WorkspaceDto> workspaces) throws OperationException
                                                   new ConfirmCallback() {
                                                       @Override
                                                       public void accepted() {
-                                                          startWorkspacePresenter.show(workspaces, callback);
+//                                                          startWorkspacePresenter.show(workspaces, callback);
```

Comment:
Because I'm not sure that this behavior is ok. It will be easier to restore old behavior if we need.